index.html

```diff
@@ -5,9 +5,12 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="styles/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
     <title>Flavio Benedito</title>
 </head>
 <body>
-    
+    <header>Flavio Benedito</header>
```

Comment:
Can you add Flavio Benedito as a heading title as well?
```suggestion
    <header>
        <h1>Flavio Benedito</h1>
    </header>
```